src/widgets/CapabilityText.tsx

```diff
@@ -70,30 +70,48 @@ export class CapabilityText {
             [WidgetKind.Room]: {
                 [EventDirection.Send]: _td("Change the topic of this room"),
                 [EventDirection.Receive]: _td("See when the topic changes in this room"),
```

Comment:
Could we at least use different verbs on this side, or is that too developer-focused...? For example, the receive case could be "Listen for ..." while the read case is "See ...".

Reply:
ideally we avoid developer language - I was hoping that the distinction between "see when it changes" and "see the current" would be enough

Reply:
Hmm, as someone _is_ aware (mainly by looking at this PR), I don't like it, but ~no one else will know the spectrum of what is possible.

I think a quick Design opinion on this copy question would help. I'll change my review to approval, since the text can be changed later, but please do at least ask for the opinion.